File: code_examples/4-Functions/2-Functions_with_a_Parameter/C/main.c
// Copyright (c) 2020 Mr. Coxall All rights reserved.
//
// Created by: Mr. Coxall
// Created on: Sep 2020
// This program calculates the area of circle

#include <stdio.h>
#include <math.h>

void calculateArea(int radius) {
    // this function calculates the area of circle

    // process
    int area = M_PI * radius ** 2;

    // output
    printf("The area is %d cm²\n", area);
}

int main() {
    // input
    printf("Enter the radius of a circle (cm): ");
    int radius;
    scanf("%d", &radius);
    printf("\n");

    // call functions
    calculateArea(radius);

    printf("\nDone.\n");
    return 0;
}

File: code_examples/4-Functions/2-Functions_with_a_Parameter/JavaScript/main.js
/**
 * Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program calculates the area of circle
 */

const prompt = require('prompt-sync')();

function calculateArea(radius) {
  // this function calculates the area of circle
  
  // process
  const area = Math.PI * radius ** 2;

  // output
  console.log(`The area is ${area} cm²`);
}

// input
const radius = parseInt(prompt("Enter the radius of a circle (cm): "));
console.log();

// call functions
calculateArea(radius);

console.log("\nDone.");

